webpack.config.js

```diff
@@ -1,6 +1,9 @@
 const path = require('path')
 const webpackConfig = require('@nextcloud/webpack-vue-config')
 
+// Let webpack determine automatically where it's located
+webpackConfig.output.publicPath = 'auto'
```

Comment:
Why is this not enough ?
https://github.com/nextcloud-libraries/webpack-vue-config/blob/master/webpack.config.js#L48